src/strife/r_main.c

```diff
@@ -915,6 +957,8 @@ void R_SetupFrame (player_t* player)
 //
 void R_RenderPlayerView (player_t* player)
 {	
+    extern void R_InterpolateTextureOffsets (void);
```

Comment:
1. Function prototypes don't require `extern`.
2. It should be in some header where it's visible to both this code and the actual definition. That way if someone changes it you will get a warning instead of a nasty crash.

Reply:
Fixed by https://github.com/fabiangreffrath/crispy-doom/pull/998/commits/cd54aafd94b0fc2708424094ea5b5e044e5b607d. Note: the other games have the same issue.